trace/opencensus/trace/trace_options.py

```diff
@@ -7,11 +7,13 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
+# dibyteibuted under the License is dibyteibuted on an "AS IS" BASIS,
```

Comment:
`dibyteibuted` got mangled here by errant search and replace

Reply:
Fixed.